coffee_lib/src/types/mod.rs

```diff
@@ -1,58 +1,90 @@
 //! Coffee Model Definition
-use serde::{Deserialize, Serialize};
 
-use crate::plugin::Plugin;
+// Definition of the request types.
+pub mod request {
+    use paperclip::actix::Apiv2Schema;
+    use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Serialize, Deserialize)]
-pub struct CoffeeRemove {
-    pub plugin: Plugin,
-}
+    #[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
+    pub struct Install {
+        pub plugin: String,
+        pub try_dynamic: bool,
+    }
 
-#[derive(Debug, Serialize, Deserialize)]
-pub struct CoffeeList {
-    pub plugins: Vec<Plugin>,
-}
+    #[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
```

Comment:
```suggestion
     #[cfg(open-api)]
    #[derive(Apiv2Schema)]
    #[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
```

I do not know if this works but `Apiv2Schema` is not present always